### PR TITLE
adapt to actuation cmd/status as control msg

### PR DESCRIPTION
**Pull request by taikitanaka3**
## Description

- precious
```
 - autowre_vehicle_msgs/ActuationCommandStamped 
 - /vehicle/status/steering
- /vehicle/status/twist 
```
- change
```
- autowre_control_msgs/ActuationCommandStamped 
- /vehicle/status/steering_status 
- /vehicle/status/velocity_status
- /control/command/emergency_cmd
```

## Review Procedure

depends
need to merge this first
https://github.com/tier4/autoware_iv_msgs/pull/66

add below to pacmod_interface.launch
ros2 launch pacmod_interface pacmod_interface.launch.xml vehicle_model:=lexus
```
  <!-- Global parameters -->
  <include file="$(find-pkg-share autoware_global_parameter_loader)/launch/global_params.launch.py">
    <arg name="vehicle_model" value="$(var vehicle_model)"/>
  </include>
```

ros2 launch raw_vehicle_cmd_converter raw_vehicle_converter.launch.xml [INFO] [launch]
ros2 topic list
```
/control/command/actuation_cmd
/control/command/control_cmd
/control/command/emergency_cmd
/control/command/gear_cmd
/control/command/hazard_lights_cmd
/control/command/turn_indicators_cmd
/control/raw_vehicle_cmd_converter/debug/steer_pid
/diagnostics
/localization/kinematic_state
/pacmod/as_rx/accel_cmd
/pacmod/as_rx/brake_cmd
/pacmod/as_rx/raw_steer_cmd
/pacmod/as_rx/shift_cmd
/pacmod/as_rx/steer_cmd
/pacmod/as_rx/turn_cmd
/pacmod/can_rx
/pacmod/can_tx
/pacmod/parsed_tx/accel_rpt
/pacmod/parsed_tx/brake_rpt
/pacmod/parsed_tx/global_rpt
/pacmod/parsed_tx/shift_rpt
/pacmod/parsed_tx/steer_rpt
/pacmod/parsed_tx/turn_rpt
/pacmod/parsed_tx/wheel_speed_rpt
/pacmod_dynamic_parameter_changer/debug/parameter
/parameter_events
/rosout
/vehicle/engage
/vehicle/status/actuation_status
/vehicle/status/control_mode
/vehicle/status/gear_status
/vehicle/status/hazard_lights_status
/vehicle/status/steering_status
/vehicle/status/turn_indicators_status
/vehicle/status/velocity_status
```

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by taikitanaka3** [comment]
@takayuki5168 
can you take a look at ?

**Comment by takayuki5168** [comment]
@taikitanaka3 Don't you modify pacmod_interface as well?

**Comment by taikitanaka3** [comment]
@takayuki5168 
OK I will add to this commit

**Comment by taikitanaka3** [comment]
@takayuki5168 
Done at cabba85 can you  take a look at this?
 https://github.com/tier4/AutowareArchitectureProposal.iv/pull/646#issue-1053336772

**Comment by taikitanaka3** [comment]
b122d2a is a optional commit for other sttus fix

**Review by tkimura4 — APPROVED** [review]
LGTM

File: localization/twist_estimator/gyro_odometer/launch/gyro_odometer.launch.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <launch>
   <arg name="use_twist_with_covariance" default="true" description="use `input_vehicle_twist_with_covariance_topic` instead of 'input_vehicle_twist_topic' as input topic"/>
-  <arg name="input_vehicle_twist_topic" default="/vehicle/status/twist" description="input twist topic name from vehicle"/>
+  <arg name="input_vehicle_twist_topic" default="/vehicle/status/velocity_status" description="input twist topic name from vehicle"/>
```
Comment (taikitanaka3) [inline localization/twist_estimator/gyro_odometer/launch/gyro_odometer.launch.xml:3]: @YamatoAndo 
https://star4.slack.com/archives/C02JH3FME8N/p1636428102198500?thread_ts=1636427333.193300&cid=C02JH3FME8N
こちら想定通りか見ていただけないでしょううか？

Comment (YamatoAndo) [reply localization/twist_estimator/gyro_odometer/launch/gyro_odometer.launch.xml:3]: すみません、gyro odometerのinputはTwist&IMUを想定しているので
この部分については修正を取り下げてもらえますでしょうか？ :bow:

Comment (taikitanaka3) [reply localization/twist_estimator/gyro_odometer/launch/gyro_odometer.launch.xml:3]: 了解です

Comment (taikitanaka3) [reply localization/twist_estimator/gyro_odometer/launch/gyro_odometer.launch.xml]: @YamatoAndo 
2e8cd44で戻しました

File: sensing/preprocessor/pointcloud/pointcloud_preprocessor/src/concatenate_data/concatenate_data_nodelet.cpp [hunk 2]
```diff
@@ -133,7 +133,7 @@ PointCloudConcatenateDataSynchronizerComponent::PointCloudConcatenateDataSynchro
     auto twist_cb = std::bind(
       &PointCloudConcatenateDataSynchronizerComponent::twist_callback, this, std::placeholders::_1);
     sub_twist_ = this->create_subscription<autoware_auto_vehicle_msgs::msg::VelocityReport>(
-      "/vehicle/status/twist", rclcpp::QoS{100}, twist_cb);
+      "/vehicle/status/velocity_status", rclcpp::QoS{100}, twist_cb);
```
Comment (taikitanaka3) [inline sensing/preprocessor/pointcloud/pointcloud_preprocessor/src/concatenate_data/concatenate_data_nodelet.cpp:136]: @YamatoAndo 
こちらもチェックお願いします

File: localization/twist_estimator/gyro_odometer/launch/gyro_odometer.launch.xml (hunk 1 above)
Comment (takayuki5168) [reply localization/twist_estimator/gyro_odometer/launch/gyro_odometer.launch.xml]: @YamatoAndo
(一応誤解が無いよう説明します)
まず前提として、/vehicle/status/twistというトピックは消えて/vehicle/status/velocity_statusという名前になります。
gyro odometerはvehicleのトピック(/vehicle/status/*)をそのまま受け取らずlocalization内でvelocity reportからtwistに変換されたものを受け取るということですよね。
ここでは一旦/vehicle/status/twistのままにしたので、後に安藤さんが適切なトピック名に変更してくださるようお願いします！
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/646/commits/2e8cd44ed2ca76cd2103e0fd3e6ecf84dcb376a7

File: sensing/preprocessor/pointcloud/pointcloud_preprocessor/src/concatenate_data/concatenate_data_nodelet.cpp (hunk 2 above)
Comment (taikitanaka3) [reply sensing/preprocessor/pointcloud/pointcloud_preprocessor/src/concatenate_data/concatenate_data_nodelet.cpp:136]: こちらはvelocity reportを使用しているので大丈夫そうですね。

**Pull request merged**